controller/controller_test.go

```diff
@@ -206,7 +206,14 @@ func TestNewDriverTasks(t *testing.T) {
 				Source:   "source",
 				VarFiles: []string{},
 				Condition: &config.ServicesConditionConfig{
-					Regexp: config.String(""),
+					config.ServicesMonitorConfig{
+						Regexp: config.String(""),
+					},
+				},
+				SourceInput: &config.ServicesSourceInputConfig{
+					config.ServicesMonitorConfig{
+						Regexp: config.String(""),
+					},
```

Comment:
honestly, I think I just added the source_input "services" tests here because I saw condition "services" being tested. I'll re-evaluate this in the `source_input "consul-kv" feature branch`

Reply:
@lornasong actually, I added this because like ServicesConditionConfig, ServicesSourceInputConfig is set to an empty default when finalized. So I believe this is why it's required.